src/modules/landing_target_estimator/LandingTargetEstimator.h

```diff
@@ -64,12 +64,16 @@
 #include <lib/conversion/rotation.h>
 #include "KalmanFilter.h"
 
+#include <uORB/topics/offboard_control_mode.h>
+#include <uORB/topics/vehicle_status.h>
+#include <uORB/topics/actuator_controls.h>
```

Comment:
These are red flags :smiley: 
Why would the landing target estimator need any of these topics? Looks like separation of concern is violated if it does.